### PR TITLE
feat: add true heading and track

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of Changes

This PR adds true heading and true track into the current ADIRS data.

## References

FBW-34-01

## Additional context

Discord username (if different from GitHub): Erick [Z-6]#6484

## Testing instructions

Make sure that the ND still accurately represents the heading and track on the display.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/systems/src/navigation/adirs.rs [hunk 1]
```diff
@@ -1031,6 +1053,8 @@ impl InertialReference {
             SignStatus::NoComputedData
         };
 
+        self.true_heading
+            .set_value(simulator_data.true_heading, ssm);
```
Comment (tracernz) [inline src/systems/systems/src/navigation/adirs.rs:1057]: The heading should become available prior to full allignment, whereas the ssm you're using here does not.

File: src/systems/systems/src/navigation/adirs.rs [hunk 2]
```diff
@@ -1056,6 +1080,16 @@ impl InertialReference {
...
             ssm,
         );
+
+        self.true_track.set_value(
+            if ground_speed_above_minimum_threshold {
+                simulator_data.true_track
+            } else {
+                simulator_data.true_heading
+            },
+            ssm,
```
Comment (tracernz) [inline src/systems/systems/src/navigation/adirs.rs:1090]: This should use the ssm of true heading as well below 50 knots.

**Review by tracernz — CHANGES_REQUESTED** [review]

**Review by davidwalschots — APPROVED** [review]
🚀 Let's get this in!

**Review by tracernz — APPROVED** [review]
🚀

**Review by beheh — APPROVED** [review]
Looks good! It would be nice at some point to properly implement the refresh interval, but applies to all ADIRS values.

**Comment by aguther** [comment]
Tested in sim and still working:
image

Looking at the code changes, there are no users of the new variables. This is to be expected.

**Pull request merged**